### PR TITLE
Update create-configuration.md

### PR DESCRIPTION
**Pull request by matteo-bottini**
Rephrased sentence about how is composed a configuration.

### Description of your changes

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: docs/getting-started/create-configuration.md [hunk 1]
```diff
@@ -27,9 +27,8 @@ installed. It will consist of three files:
 Crossplane can create a configuration from any directory with a valid
 `crossplane.yaml` metadata file at its root, and one or more XRDs or
 Compositions. The directory structure does not matter, as long as the
-`crossplane.yaml` file is at the root. Note that a configuration need not
-contain one XRD and one composition - it could include only an XRD, only a
-composition, several compositions, or any combination thereof.
+`crossplane.yaml` file is at the root. 
+Note: a configuration is not necessarily composed by just one XRD and just one composition - it could include only an XRD, only a composition, several compositions, or any combination thereof.
```
Comment (negz) [inline docs/getting-started/create-configuration.md:31]: Please wrap at 80 chars to match the rest of the document.

**Comment by negz** [comment]
@matteo-bottini Thanks! You'll need to sign the commit per the DCO check.

**Comment by negz** [comment]
@matteo-bottini I'm seeing DCO is still missing per https://github.com/crossplane/crossplane/pull/3212/checks?check_run_id=7831245700

**Comment by matteo-bottini** [comment]
> 

Yes, sorry, I'll take care of this.

**Comment by negz** [comment]
I'm going to close this one out - let me know if you're still interested in this fix and we can reopen once we have the commit signed.

**Pull request closed without merging**